### DISCUSSION TITLE
Add Turbowarp Settings for Turbowarp Viewer

### DISCUSSION THREAD

**Discussion by rgantzos** [ScratchAddons/ScratchAddons#3970] [Ideas] [answered]
Here's an idea: with the feature that was added so that you can replace the Scratch viewer with the Turbowarp viewer, you could add in a way to change the settings. For example, the interpolation settings, fps settings, and more.

If possible, I would love to help work on this idea.

**Comment by cobaltt7** [comment]
Not in the scope of SA


________________________________

**Comment by WorldLanguages** [comment, marked as answer]
Not all of those settings can be controlled in a turbowarp iframe.

**Reply by Samq64** [reply]
Most of them work if the creator of the project added an advanced settings stage comment.

**Reply by CST1229** [reply]
The more used features of turbowarp (interpolation, fps, clone limit, remove limits, stage size) can be all controlled from the URL I think.

**Reply by kccuber-scratch** [reply]
> The more used features of turbowarp (interpolation, fps, clone limit, remove limits, stage size) can be all controlled from the URL I think.

addons also work, i think

**Reply by WorldLanguages** [reply]
@kccuber-scratch Addons are already supported (enable pause button, refresh, go to turbowarp player and see it has pause button)
